atomic_reactor/plugins/post_generate_maven_metadata.py

```diff
@@ -34,7 +33,9 @@ def __init__(self, workflow):
         super(GenerateMavenMetadataPlugin, self).__init__(workflow)
         self.source_url_to_artifacts = {}
 
-    def get_remote_source_files(self, download_queue):
+    def get_remote_source_files(
+            self, download_queue: Sequence[DownloadRequest]
+    ) -> List[Dict[str, Collection[str]]]:
```

Comment:
I'd go with `List[dict]` or `List[Dict[str, Any]]`, the fact that all the values are collection of strings is more misleading than helpful (one value is a string, the other is a dict[str, something], both of which happen to be collection of strings)

Reply:
updated type annotations to `List[Dict[str, Any]]`